test/suites/static_analysis.sh

```diff
@@ -26,6 +26,13 @@ test_static_analysis() {
       echo "shellcheck not found, shell static analysis disabled"
     fi
 
+    # golangci-lint
+    if command -v golangci-lint >/dev/null 2>&1; then
+      golangci-lint run --disable-all -E deadcode
+    else
+      echo "shellcheck not found, shell static analysis disabled"
```

Comment:
"shellcheck" is probably copy-n-pasto ;)

Reply:
Whoopsie!